app/services/scrapers/case.rb

```diff
@@ -10,8 +10,16 @@ def perform
       # TODO: Capture results and send via email or slack message
       Scrapers::WarrantsView.perform
       counties.each do |county|
-        Scrapers::NewCases.perform(county, days_ago: days_ago)
-        Scrapers::HighPriority.perform(county, days_ago: days_ago, days_forward: days_forward)
+        if OscnScraper::Requestor::DAILING_FILINGS_COUNTIES.include? county
+          Scrapers::NewCases.perform(county, days_ago: days_ago)
+          Scrapers::HighPriority.perform(county, days_ago: days_ago, days_forward: days_forward)
+        else
+          date_range = (DateTime.now.to_date - days_ago...DateTime.now.to_date)
+          date_range.each do |date|
+            Scrapers::DailyFilingsAltCounties.perform(county, date, enqueue: false)
+          end
+          Scrapers::HighPriority.perform(county, include_event_updates: false)
+        end
```

Comment:
It might easier to understand if you use the counties table scope `counties.ocis` and `counties.kp` rather than a constant outside of the project.

```ruby
enum rms: {
    ocis: 'OCIS',
    kp: 'KP'
}
```